stdlib/3/os/__init__.pyi

```diff
@@ -121,6 +121,8 @@ if sys.version_info >= (3, 6):
 _PathType = Union[bytes, Text]
 
 if sys.version_info >= (3, 6):
+    _PathType = Union[bytes, Text, PathLike]
```

Comment:
If you're importing this from other modules, why is it private? (I.e., why the leading underscore?)